include/LLVMSPIRVOpts.h

```diff
@@ -193,6 +193,14 @@ class TranslatorOpts {
     ReplaceLLVMFmulAddWithOpenCLMad = Value;
   }
 
+  void setUseOpenCLExtInstructionsForLLVMMathIntrinsic(bool Value) noexcept {
```

Comment:
nit regarding `noexcept`:
LLVM itself prohibits [C++ exceptions](https://llvm.org/docs/CodingStandards.html#do-not-use-rtti-or-exceptions) and they are turned off in CMakeLists by compiler flag `-fno-exceptions`.
If we use exceptions in translator, then it would be better to stop doing it and start using the compiler flag `-fno-exceptions` as well.